script/style/checkstyle.xml

```diff
@@ -40,6 +40,9 @@
     <module name="NoFinalizer" />
     <module name="OneStatementPerLine" />
     <module name="ParameterAssignment" />
+    <module name="RequireThisCheck">
+      <property name="checkMethods" value="false"/>
```

Comment:
leaving the `checkMethods` part of this as false for now, we can flip to true and audit again in the future but this took enough busy time 😬